### PR TITLE
Dont lambda

### PR DESCRIPTION
**Pull request by hadasz**
**Related Issue**  
Supports https://github.com/ConsenSys/web3studio-bootleg/issues/129

**What does this PR do?**  
This PR is the setup of a lambda function that responds to events from AWSAPIGateway. There are two endpoints, one that displays an array of integers, and one that gets rejected because it goes through an authorizer function that isn't implemented yet. The purpose is to prepare for creating an authorizer function and API endpoint. It contains a precursor to an integration test. 
`npm run integration-test`
In browser, 
    -go to http://localhost:3000/getrejected - confirm rejection message displays in browser
     -go to http://localhost:3000/getintarray - confirm int array displays in browser
**Description of Changes**  
-created dont-crypto folder
-set up template haskell package and stuff needed to deploy lambda
**What gif most accurately describes how I feel towards this PR?**  
Example of a gif

### CODE REVIEW

File: packages/dont-crypto/test/Spec.hs [hunk 1]
```diff
@@ -0,0 +1,2 @@
+main :: IO ()
+main = putStrLn "Test suite not yet implemented"
```
Comment (breakpointer) [inline packages/dont-crypto/test/Spec.hs]: Hoping to see more here in the future 😸

File: packages/dont-crypto/app/Main.hs [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{-# LANGUAGE OverloadedStrings #-}
+import AWSLambda.Events.APIGateway
+import Control.Lens
+import Data.Aeson
+import Data.Aeson.Embedded
+main = apiGatewayMain handler
+
+handler :: APIGatewayProxyRequest (Embedded Value) -> IO (APIGatewayProxyResponse (Embedded [Int]))
```
Comment (breakpointer) [inline packages/dont-crypto/app/Main.hs]: Dang. So cool. Can't wait to see more Haskell

File: packages/dont-crypto/README.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# dont-crypto
+
+This is the directory where the dont cryptography implementation will live
```
Comment (breakpointer) [inline packages/dont-crypto/README.md]: Would you mind adding some setup steps so I can try this out? I'm not sure about the `cabal` stuff or what I'd need to build/deploy haskell serverless.

File: packages/dont-crypto/LICENSE [hunk 4]
```diff
@@ -0,0 +1,30 @@
+Copyright Author name here (c) 2019
```
Comment (barlock) [inline packages/dont-crypto/LICENSE]: Should this be our Apache 2 license?

File: packages/dont-crypto/package-lock.json [hunk 5]
```diff
@@ -0,0 +1,3793 @@
+{
```
Comment (barlock) [inline packages/dont-crypto/package-lock.json]: I think this was checked in on accident?

Comment (hadasz) [reply packages/dont-crypto/package-lock.json]: yea, good catch, i'll remove it

File: packages/dont-crypto/LICENSE (hunk 4 above)
Comment (hadasz) [reply packages/dont-crypto/LICENSE]: yea, it should be. I'll update it

**Pull request merged**